XenAdmin/Diagnostics/Problems/ConnectionProblem/VmprActivatedWarning.cs

```diff
@@ -55,7 +55,9 @@ public override string Description
 
         public override Uri UriToLaunch
         {
-            get { return new Uri(InvisibleMessages.VMPR_DEPRECATION_URL); }
+            get {
+                return new Uri(XenAdmin.Core.Registry.LinkLabelHidden ? "" : InvisibleMessages.VMPR_DEPRECATION_URL); 

```

Comment:
I think that the Uri(string) constructor will raise a UriFormatException if the string is empty.
We cound instead do:
return XenAdmin.Core.Registry.LinkLabelHidden ? null : new Uri(InvisibleMessages.VMPR_DEPRECATION_URL);
Also classes WarningWithInformationUrl and ProblemWithInformationUrl need to cope with UriToLaunch==null.